tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/AwsLambda/LambdaSqsEventTriggerFixture.cs

```diff
@@ -0,0 +1,142 @@
+                              }
+                            ]
+                          }
+                          """;
+            EnqueueLambdaEvent(sqsJson);
+        }
+
+        public void EnqueueSqsEventWithDTHeaders(string traceId, string spanId)
+        {
+            var sqsJson = $$"""
+                          {
+                            "Records": [
+                              {
+                                "messageId": "19dd0b57-b21e-4ac1-bd88-01bbb068cb78",
+                                "receiptHandle": "MessageReceiptHandle",
+                                "body": "Hello from SQS!",
+                                "attributes": {
+                                  "ApproximateReceiveCount": "1",
+                                  "SentTimestamp": "1523232000000",
+                                  "SenderId": "123456789012",
+                                  "ApproximateFirstReceiveTimestamp": "1523232000001"
+                                },
+                                "messageAttributes": {
+                                  "Test": {
+                                    "Type": "String",
+                                    "StringValue": "TestString"
+                                  },
+                                  "TestBinary": {
+                                    "Type": "Binary",
+                                    "BinaryValue": "VGVzdEJpbmFyeQ=="
+                                  },
+                                   "traceparent": {
+                                    "Type": "String",
+                                    "StringValue": "{{GetTestTraceParentHeaderValue(traceId, spanId)}}"
+                                  },
+                                  "tracestate": {
+                                    "Type": "String",
+                                    "StringValue": "{{GetTestTraceStateHeaderValue(spanId)}}"
+                                  }
+                                },
+                                "md5OfBody": "7b270e59b47ff90a553787216d55d91d",
+                                "eventSource": "aws:sqs",
+                                "eventSourceARN": "arn:{partition}:sqs:{region}:123456789012:MyQueue",
+                                "awsRegion": "us-west-2"
+                              }
+                            ]
+                          }
+                          """;
+            EnqueueLambdaEvent(sqsJson);
+        }
+    }
+
+    public class LambdaSqsEventTriggerFixtureNet6 : LambdaSqsEventTriggerFixtureBase
+    {
+        public LambdaSqsEventTriggerFixtureNet6() : base("net6.0", false, true) { }
+    }
+
+    public class AsyncLambdaSqsEventTriggerFixtureNet6 : LambdaSqsEventTriggerFixtureBase
+    {
+        public AsyncLambdaSqsEventTriggerFixtureNet6() : base("net6.0", true, true) { }
+    }
+
+    public class LambdaHandlerOnlySqsTriggerFixtureNet6 : LambdaSqsEventTriggerFixtureBase
```

Comment:
Do we need to have all of these permutations? I think most other tests are only using the sync / async and net6 / net8 versions, not worrying with the HandlerOnly stuff.

Reply:
I was unsure if we needed them so I kept them, but I can remove them.

Reply:
Probably best to remove them to try to contain the number of tests.